src/frontend/src/common/queries/locations.ts

```diff
@@ -50,6 +42,45 @@ export function useLocations(): UseQueryResult<LocationsResponse, unknown> {
   });
 }
 
+const fetchPathogenDepthLocations = (): Promise<LocationsResponse> => {
```

Comment:
Created a new query because mpox needs the full locations list for things like sample upload but needs the list at the division level for the nextstrain tree creation modal.